### PR TITLE
zbar .so library for pyzbar (reading QR codes)

### PR DESCRIPTION
**Pull request by mrakitin**
http://zbar.sourceforge.net/

### CODE REVIEW

**Comment by jrmlhermitte** [comment]
looks good to me.

File: recipes-tag/zbar/build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+./configure --prefix=$PREFIX --disable-video --without-imagemagick --without-gtk --without-qt --without-python
```
[the remark this answers is not in the document]
Comment (jrmlhermitte) [reply recipes-tag/zbar/build.sh]: thanks, looks good

**Comment by jrmlhermitte** [comment]
looks good to me

**Comment by mrakitin** [comment]
Thanks for reviewing and merging :)

File: recipes-tag/zbar/build.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (mrakitin) [reply recipes-tag/zbar/build.sh]: For future us: https://conda.io/docs/user-guide/tasks/build-packages/environment-variables.html?highlight=PREFIX

**Pull request merged**